### DISCUSSION TITLE
Row checkbox does not show

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7836] [Help] [answered]
I check the "enable row selection" but the checkbox does not appear :
<img width="929" alt="image" src="https://user-images.githubusercontent.com/28592069/190979503-9e2f5626-6a0a-4080-848e-5bed1fb77fef.png">

Even if I create a brand new table, it does not work...

**Comment by melohagan** [comment, marked as answer]
This looks like a custom query? Related issue: https://github.com/Budibase/budibase/issues/7348

**Reply by NicolasFabre81** [reply]
Yes it is, I am obliged to use it because date filters does'nt work, see [#7893](https://github.com/Budibase/budibase/discussions/7893)
